uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -525,6 +530,7 @@ public synchronized void updateCurrentStatus() {
       List<String> liveInstances = HelixUtils.liveInstances(_helixManager);
       currAvailableControllerList.addAll(liveInstances);
 
+      int assignedCount = 0;
```

Comment:
Can we rename this to assignedControllerCount(er) so we can also monitor assigned worker in the future.